src/ucp/wireup/wireup.c

```diff
@@ -535,6 +536,9 @@ ucp_wireup_process_request(ucp_worker_h worker, ucp_ep_h ep,
                              worker, ep, worker->context);
                 }
             }
+        } else if (ucp_ep_config_err_mode_check_mismatch(ep, msg->err_mode)
```

Comment:
maybe do it unconditionally (without else) to catch bug in ucp_worker_create_ep ?

Reply:
done